packages/core/utils/src/sanitize/index.ts

```diff
@@ -26,6 +26,9 @@ export interface SanitizeFunc {
 
 const createContentAPISanitizers = () => {
   const sanitizeInput: SanitizeFunc = (data: unknown, schema: Model, { auth } = {}) => {
+    if (!schema) {
```

Comment:
We could actually improve this further and try to detect if it's a valid schema, in case the user is attempting to pass in the wrong type of object (like if they pass in the string uid instead of the actual model)